main.go

```diff
@@ -143,6 +145,25 @@ func main() {
 		return
 	}
 
+	cli.CdnRedirectPlaybackPct = make(map[string]float64)
+
+	for _, redirect := range cli.CdnRedirectPlaybackIDs {
+		elements := strings.Split(redirect, ":")
+		if len(elements) == 2 {
+			percentage, err := strconv.ParseFloat(elements[1], 64)
+			if err != nil || percentage > 100.0 || percentage < 0.0 {
+				glog.Fatalf("Invalid config %s - should be between 0.0 and 100.00", redirect)
+			} else {
+				cli.CdnRedirectPlaybackPct[elements[0]] = percentage / 100.0
```

Comment:
Oh I see... I was confused on whether this was a 0-100 or 0-1 value. Might be worth a comment in the `CdnRedirectPlaybackPct` explaining that it's provided as 0-100 but stored as 0-1 for the load balancing logic